lambda/helper/python/comprehendHelper.py

```diff
@@ -0,0 +1,454 @@
+import boto3
+from botocore.exceptions import ClientError
+import json
+import threading
+from helper import S3Helper
+
+
+class ComprehendHelper:
+
+    def getNumOfPages(self,
+                      textract):
+        return textract[0]['DocumentMetadata']['Pages']
+
+    #
+    # this function iterates and extracts all textual elements, aka blocks, that were
+    # identified by Textract in each page.  The result is organized as a list of strings,
+    # each containing the concatenated text of a given page.
+    #
+    def extractTextByPages(self,
+                           textract,
+                           rawPages,
+                           numOfPages):
+
+        # current service limit of Comprehend per page
+        MAX_COMPREHEND_UTF8_PAGE_SIZE = 5000
```

Comment:
NIT : Can we constants on the top of the file? Makes it easier to read.

Reply:
certainly